sandbox/src/mocks/r4/NHSDigital-OperationOutcome.json

```diff
@@ -0,0 +1,24 @@
+{
+  "resourceType": "OperationOutcome",
+  "meta": {
+    "profile": [
+      "https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome"
+    ],
+    "lastUpdated": "2022-03-01T10:00:00.42Z"
+  },
+  "issue": [
+    {
+      "severity": "error",
+      "code": "forbidden",
+      "diagnostics": "Example diagnostics message.",
+      "details": {
+        "coding": [
+          {
+            "system": "https://fhir.nhs.uk/CodeSystem/http-error-codes",
+            "code": "REC_BAD_REQUEST"
+          }
+        ]
+      }                      
+    }
+  ]
+}
```

Comment:
I think this file can be removed as the examples need to be more specific now. We want the examples to reflect the http code and the issue.details.coding.code:
So if we're adding e.g. an example for 415, we'd need an NHSDigital-OperationOutcome-415.json with REC_UNSUPPORTED_MEDIA_TYPE code